mantidproject/mantid
# Indirect - CalculateMonteCarloAbsorption - Documentation and Workflow Diagram

This PR adds documentation and a workflow diagram for the CalculateMonteCarloAbsorption algorithm.

**To test:**
1. Read through the \.rst file and check features have been described and example works as expected.
2. Ensure workflow diagram matches workflow of algorithm.



Fixes #20787 . 

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
